src/core/components/stateful.spec.ts

```diff
@@ -189,6 +205,14 @@ describeComponent('stateful component', [
         expect((<DummyStatefulComponent> component.ctrl.childComponents()[2]).bar).toBe(42);
     });
 
+    it('throws error if the state is not serializable', () => {
+        const component = tester.createComponent<DummyNotSaveableStatefulComponent>(
+            DummyStatefulComponent.asView().template
+        );
+
+        expect(component.ctrl.saveState).toThrow();
```

Comment:
This always fails. Changed to `expect(() => component.ctrl.saveState()).toThrow();`

Reply:
And fixed to `DummyNotSaveableStatefulComponent.asView().template` above